.code-intelligence/campaigns/parse_query-20e3604d.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "parse_query-20e3604d",
```

Comment:
The numeric identifiers are autogenerated but I'm not sure if they're required, I'm tempted to just remove them and see what happens.